apps/taxes/src/modules/channel-configuration/ui/channel-section.tsx

```diff
@@ -0,0 +1,49 @@
+import { actions, useAppBridge } from "@saleor/app-sdk/app-bridge";
+import { Text } from "@saleor/macaw-ui/next";
+import { Section } from "../../ui/app-section";
+import { ChannelList } from "./channel-list";
+
+const Intro = () => {
+  const appBridge = useAppBridge();
+
+  const redirectToTaxes = () => {
+    appBridge.appBridge?.dispatch(actions.Redirect({ to: "/taxes/channels" }));
+  };
+
+  return (
+    <Section.Description
+      title="Available channels"
+      description={
+        <>
+          This table displays all the channels configured to use the tax app as the tax calculation
+          method.
+          <br />
+          <br />
+          You can change the tax configuration method for each channel in the{" "}
+          <Text
```

Comment:
What you see here is a literal copy of it. I did it because `TextLink` works for two cases:
1. External links
2. In-app links

Whereas, here we have a third case: an in-dashboard link. Unfortunately `TextLink` didn't work for me.

Reply:
Can you modify shared TextLink to handle this use case?

If you want to avoid a bigger refactor caused by it, let's create an issue ticket for that and resolve this convo.

Reply:
I will add it but after I finish this whole thing